frontend/src/components/poster/Poster.jsx

```diff
@@ -6,7 +6,7 @@ function Poster({ poster }) {
   const imageUrl = "https://image.tmdb.org/t/p/original";
   return (
     <img
-      className="poster__img"
+      className="poster__img poster poster"
```

Comment:
Twice poster

Reply:
argh, it's some test residue ! Removed !